operations/service.go

```diff
@@ -55,6 +56,10 @@ func Service() cli.Command {
 					Name:  localQueueFlag,
 					Usage: "uses a locally-backed queue rather than MongoDB",
 				},
+				cli.BoolFlag{
+					Name:  disableLocalLoggingFlag,
+					Usage: "disable logging to stdout",
```

Comment:
Did you want to be able to set it from the environment too? (if so you could specify an `EnvVar`)

Reply:
Good point, added